tests/test_minhash.py

```diff
@@ -231,6 +231,24 @@ def test_dayhoff(track_abundance):
     assert mh_protein.hashes != mh_dayhoff.hashes
 
 
+def test_dayhoff_2(track_abundance):
+    mh = MinHash(0, 7, scaled=1, is_protein=True, dayhoff=True,
```

Comment:
I am wondering why is_protein = True given this code block:

https://github.com/sourmash-bio/sourmash/blob/039345686043001b84c1494dcc43c2247efdfc72/src/sourmash/minhash.py#L195-L196

Reply:
good catch.

Reply:
(I wasn't sure either, so I wrote it and then it seemed to work so 🤷 and now you've pointed out the code for why it does 😁 )